.eslintrc

```diff
@@ -22,6 +22,7 @@
 		"plugin:prettier/recommended"
 	],
 	"rules": {
-		"prettier/prettier": "warn"
+		"prettier/prettier": "warn",
+		"roblox-ts/lua-truthiness": "off"
```

Comment:
Why would you turn this off?

Reply:
I don't know I forgot

Reply:
Oh- that's because there's a rule that warns against typical ternary usage because "" and a few other things are falsey. I think we should leave it on imo